hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -300,6 +300,8 @@
   // server manager to deal with region server info
   private volatile ServerManager serverManager;
 
+  private volatile CompactionServerManager compactionServerManager;
```

Comment:
Is it possible to reuse ServerManager here? Just asking...

Reply:
I think no need now, has few duplicated codes. CompactionServer do not register to zookeeper, so compactionServerManager is more simple without zookeeper interaction.